snippets/celsius_to_fahrenheit.md

```diff
@@ -0,0 +1,19 @@
+---
+title: celsius_to_fahrenheit
+tags: conversion, beginner

```

Comment:
Tags should be `math,beginner` as we do not have a conversion category and `math` is where we usually put these kinds of snippets.